src/app/scripts/cac/map/cac-map-control.js

```diff
@@ -84,25 +45,13 @@ CAC.Map.Control = (function ($, Handlebars, cartodb, L, turf, _) {
         '&copy; <a href="http://cartodb.com/attributions">CartoDB</a>'
     ].join('');
 
-    /**
-     * Variables used for limiting to one isochrone request at a time.
-     * Unlike the planTrip request, the isochrone request cannot be handled
-     * solely via debounce, due to differences in the way the isochrone
-     * request flows through the system. There are actions that take place
-     * on this module (drawing the isochrone on the map), and then actions
-     * that take place on the sidebar module (generating the destinations),
-     * so doing the limiting correctly is more of a challenge.
-     */
-    var activeIsochroneRequest = null;
-    var pendingIsochroneRequest = null;
-
     function MapControl(options) {
         this.events = events;
         this.eventNames = eventNames;
         this.options = $.extend({}, defaults, options);
         overlaysControl = new CAC.Map.OverlaysControl();
         map = new cartodb.L.map(this.options.id, { zoomControl: false })
-            .setView(this.options.center, this.options.zoom);
+                           .setView(this.options.center, this.options.zoom);
```

Comment:
Oh, I thought maybe you'd accidentally put in an extra tab there.

For parentheses and brackets we'll regularly align with the line above, but our handling of dot methods varies. For several lines of them, I think indenting them all by some regular spacing multiple to be the easiest to read and maintain. This is fine.